Test installing all operators in repository

nfnt opened the pull request:
This also includes demo operators in package documentations.

Commit:
Test installing all operators in repository

This also includes demo operators in package documentations.

Signed-off-by: Jan Schlicht <jan@d2iq.com>

mpereira commented on kudo-test.yaml:
```diff
@@ -4,8 +4,10 @@ manifestDirs:
 - ./test/manifests/
 kubectl:
 - kudo init --crd-only
+- kudo install --skip-instance ./repository/cowsay/operator/
```
```suggestion
- kudo install --skip-instance ./repository/cassandra/3.11/operator/
- kudo install --skip-instance ./repository/cowsay/operator/
```

Commit:
Add more operators

Commit:
Actually test against Kubernetes 1.16

kensipe reviewed (approved):
merged to the latest on master... waiting for green otherwise lgtm

Commit:
Update

Commit:
Test against Kubernetes 1.16 and 1.17

Some operators require Kubernetes 1.16 and the test harness doesn't allow to skip these tests. Instead, let's just test against more recent versions of Kubernetes.

Signed-off-by: Jan Schlicht <jan@d2iq.com>

Commit:
Merge branch 'master' into nfnt/test-all-operators

Signed-off-by: Jan Schlicht <jan@d2iq.com>

nfnt commented:
Re-requesting reviews, because the removal of Kubernetes 1.15 testing was done after @kensipe's approval.

ANeumann82 reviewed (approved):
Waiting for green, but otherwise lgtm

ANeumann82 commented:
Oh, it's only DCO missing, therefore, go fo rit

nfnt commented:
I'll sort out the DCO issue when squashing.

zmalik reviewed (approved):
LGTM!
